src/Runner.Common/RunnerService.cs

```diff
@@ -70,7 +70,7 @@ protected async Task<VssConnection> EstablishVssConnection(Uri serverUrl, VssCre
 
         protected async Task RetryRequest(Func<Task> func,
             CancellationToken cancellationToken,
-            int maxRetryAttemptsCount = 5,
+            int maxAttempts = 5,
```

Comment:
The previous description was a misnomer. This value indicates maximum number of attempts (i.e. initial attempt + retries).